charts/opentelemetry-collector/templates/clusterrole.yaml

```diff
@@ -45,7 +45,7 @@ rules:
   {{- end}}
   {{- if .Values.presets.kubeletMetrics.enabled}}
   - apiGroups: [""]
-    resources: ["nodes/stats"]
+    resources: ["nodes/stats", "nodes/proxy"]
```

Comment:
I do not think `nodes/proxy` is required if you're using the metrics supplied by default by the preset, only if you add the `* _utilization` metrics. Can you confirm?

Reply:
That is correct yes

Reply:
Since the preset does not enable those metrics by default we do not need to enable this resource by default. When extending the preset with those metrics you can extend the cluster role via your values.yaml.